tests/checks/core/test_certificate_expiration.py

```diff
@@ -57,6 +57,20 @@ async def test_negative():
     assert data == {"expires": next_month.isoformat()}
 
 
+async def test_positive_bounded_maximum():
+    url = "https://fake.local"
+
+    last_year = utcnow() - timedelta(days=2365)
```

Comment:
```suggestion
    last_year = utcnow() - timedelta(days=365)
```

Also, I don't know how important it is that this is _exactly_ one year. Like if we need to consider leap years or something. But as long as this just needs to be "about a year in the past", this is probably fine.

Reply:
Oh weird, your comments showed up when I clicked merge :facepalm: 
Will take them into account in a follow up.